bagel/pheno_utils.py

```diff
@@ -19,6 +19,7 @@ def generate_context():
         "bg": "http://neurobagel.org/vocab/",
         "snomed": "https://identifiers.org/snomedct:",
         "nidm": "http://purl.org/nidash/nidm#",
+        "cogAtlas": "https://www.cognitiveatlas.org/task/id/",
```

Comment:
now that we're trying to be consistent with this, we should have a central place for namespace conventions (and also for names for properties/edges and classes/nodes). I'm thinking such a place doesn't exist yet (does it?) - would you mind creating an issue to create such a place?

Reply:
@surchs I think the relevant issue is #65, specifically the first subtask. Worth separating that into its own issue?